apps/content-publishing-api/src/api.service.ts

```diff
@@ -33,10 +34,36 @@ export class ApiService {
     @InjectRedis() private redis: Redis,
     @InjectQueue(QueueConstants.REQUEST_QUEUE_NAME) private requestQueue: Queue,
     @InjectQueue(QueueConstants.ASSET_QUEUE_NAME) private assetQueue: Queue,
+    @InjectQueue(QueueConstants.PUBLISH_QUEUE_NAME) private publishQueue: Queue,
   ) {
     this.logger = new Logger(this.constructor.name);
   }
 
+  async enqueueContent(
+    schemaId: number,
+    contentDto: OnChainContentDto,
+    onBehalfOf: string,
+  ): Promise<AnnouncementResponseDto> {
+    const jobData: IPublisherJob = {
+      id: Date.now().toString(), // timestamp will get overwritten, but guarantees id-uniqueness
```

Comment:
so the whole reason behind calculating JobId was to ensure duplications are detected in queue level to some degree. If you add this here it would cause every request to be unique which would cause duplications for the same content